### ISSUE TITLE
<HIGH> Code is auto-folding in the Cell Code Console at some point. 

### ISSUE THREAD

**Issue by kimo** [PySheets/pysheets#106]
This causes errors on running sheets.

@sangramsingh-kss @laffra

**Comment by kssd** [comment]
Yes, I've seen similar issue/inconvenience while trying to work with console.

Having some code editor options would be nice.
1. Block folding.
2. Auto indent.
3. Tab/space highlights.

**Comment by laffra** [comment]
What is the "Cell Code Console" and what do you mean with folding?

**Comment by kimo** [comment]
1. “Cell Code Console” is the area where we write python code for cells. I
made up the name. 😀

2. Folding - I mean the indentation is lost at some point and all code
becomes one-line.

**Comment by laffra** [comment]
1. Confirmed. In the code, I call it "editor," but without a label, I can see naming ambiguities arise.
2. This is the first time I have heard this. Is this when you do not have an "=" at the start of the code block?

**Comment by kimo** [comment]
2. I will pay attention now to track specific scenario when this behavior occurs.

**Comment by laffra** [comment]
I had this happening to me a few times now. In each case, you can go to the Timeline and "undo" those edits. I will debug this a bit more.
